test.js

```diff
@@ -867,20 +841,18 @@ test('getAsyncIteratorMethod provides function for AsyncIterable', () => {
     .then(step => assert.deepEqual(step, { value: 0, done: false }))
 })
 
-test('getAsyncIteratorMethod provides undefined for Generator', () =>
-  // $FlowExpectError
-  getAsyncIteratorMethod(genSampleFib()) === undefined)
+// test('getAsyncIteratorMethod provides undefined for Generator', () =>
+//   getAsyncIteratorMethod(genSampleFib()) === undefined)
+  // This doesn't seem right anyway.
```

Comment:
Ya - this is testing behavior that flow should be checking against anyhow, so this `$FlowExpectError` is intentional :)